website_sale_b2x_alt_price/i18n/es.po

```diff
@@ -0,0 +1,33 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* website_sale_b2x_alt_price
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-08-07 09:11+0000\n"
+"PO-Revision-Date: 2020-08-07 10:12+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.4\n"
+"Last-Translator: Jairo Llopis <jairo.llopis@tecnativa.com>\n"
+"Language: es_ES\n"
+
+#. module: website_sale_b2x_alt_price
+#: model:ir.model,name:website_sale_b2x_alt_price.model_product_template
+msgid "Product Template"
+msgstr "Plantilla de producto"
+
+#. module: website_sale_b2x_alt_price
+#: model_terms:ir.ui.view,arch_db:website_sale_b2x_alt_price.alt_price
+msgid "with taxes"
+msgstr "con impuestos"
```

Comment:
In a spanish e-commerce I'd except "IVA inc." o "excl. IVA". What do you think @pedrobaeza ?

Reply:
"IVA incluido" is not accurated enough, as there can be other taxes that are not IVA, so I think this one is ok.

Reply:
Ok, I didn't think of b2b scenarios